### PR TITLE
Add better west yaml instructions

### LINKED ISSUE #2: west.yaml name-allowlist in README
## Context

When updating the NCS version used, it is not enough to just update the revision for `nrf` and `zephyr`. One must also check the name-allowlist (or list of all modules) of the new NCS and zephyr to see if anything has changed.
Where the developer should look for this I do not know...

## Required steps / Implementation details

Add some more text to the readme describing this process.

### PR DESCRIPTION
**Pull request by MarkoSagadin**
## Description

This PR add better explanation on how to configure west.yml.

Closes #2

## Areas of interest for the reviewer

The readme.

## After-review steps

* Reviewer can merge and delete the branch.

[style guidelines]: https://github.com/IRNAS/irnas-guidelines-docs/blob/dev/docs/developer_guidelines.md

### CODE REVIEW

**Comment by MarkoSagadin** [comment]
Current dependencies on/for this PR:
* dev
  * **PR #3** <a href="https://app.graphite.dev/github/pr/IRNAS/irnas-zephyr-template/3" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/IRNAS/irnas-zephyr-template/3?utm_source=stack-comment).

File: README.md [hunk 1]
```diff
@@ -37,25 +41,50 @@ source zephyr/zephyr-env.sh
...
+   ([link](https://github.com/nrfconnect/sdk-nrf)). Make sure that you select
+   correct tag from selection from top-left dropdown menu.
+2. Check what repos are under `sdk-zepyhr` project's `name-allowlist`, those
+   should match the repos in `west.yaml` of your project (and this template),
+   under `sdk-zepyhr` project's `name-allowlist`. Most of the time they should
+   be commented out, but depends.
+3. Check what other repositories appear under `sdk-zepyhr` project as standalone
+   projects, they start appearing around line 100, after `NCS repositories`
+   comment. This projects should match the repos those should match the repos in
+   `west.yaml` of your project (and this template), under `sdk-nrf` project's
```
Comment (TjazVracko) [inline README.md:83]: ???

**Review by TjazVracko — CHANGES_REQUESTED** [review]
I will do some grammar fixes after this is merged.

File: README.md (hunk 1 above)
Comment (MarkoSagadin) [reply README.md]: Brain fart I pushed the correct text.

**Pull request merged**